www/src/components/incidents/Markdown.js

```diff
@@ -4,13 +4,16 @@ import {
 import {
   Anchor, Box, Drop, Markdown, Text, ThemeContext,
 } from 'grommet'
-import { Copy, TooltipContent, WithCopy } from 'forge-core'
-import Highlight from 'react-highlight.js'
-import hljs from 'highlight.js'
+import {
+  Copy, TooltipContent, WithCopy,
+} from 'forge-core'
+import hljs from 'highlight.js/lib/core'
 import { normalizeColor } from 'grommet/utils'
 import sortBy from 'lodash/sortBy'
 import { Emoji as Emojii } from 'emoji-mart'
 
+import Highlight from '../utils/Highlight'
```

Comment:
We can export this to design system as the next step as it looks like part of Markdown.

Reply:
We should create an issue to not forget about it.

Reply:
I will take care of it.

Reply:
https://linear.app/pluralsh/issue/ENG-584/export-highlight-component-to-design-system